lib/Optimizer/Transforms/LiftArrayAlloc.cpp

```diff
@@ -182,8 +219,8 @@ class AllocaPattern : public OpRewritePattern<cudaq::cc::AllocaOp> {
     if (std::distance(alloc->getUses().begin(), alloc->getUses().end()) < size)
       return false;
 
-    // Keep a scoreboard for every element in the array. Every element *must* be
-    // stored to with a constant exactly one time.
+    //  Keep a scoreboard for every element in the array. Every element *must*
```

Comment:
Can revert these changes. No need to add an extra space hee.

Reply:
done!